ui-admin/src/study/surveys/CreateSurveyModal.tsx

```diff
@@ -77,7 +80,7 @@ const CreateSurveyModal = ({ studyEnvContext, onDismiss, type }:
 
   return <Modal show={true} onHide={onDismiss} className={type === 'OUTREACH' ? 'modal-lg' : 'modal'}>
     <Modal.Header closeButton>
-      <Modal.Title>Create New {type === 'RESEARCH' ? 'Research Survey' : 'Outreach'}</Modal.Title>
+      <Modal.Title>Create new {type.toLowerCase()}</Modal.Title>
```

Comment:
nit: I find "Create new research" an awkward phrase - I think we should keep "survey" in there for at least research surveys, if not all of them.